### PR TITLE
Improve automated PR description for JetBrains auto-update

### PR DESCRIPTION
**Pull request by andreafalzetti**
## Description
Improving the "how-to-test" section

## How to
| Before | After |
|--------|-------|
|    Screenshot 2022-05-13 at 14 27 42    |     Screenshot 2022-05-13 at 14 26 16 |

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by andreafalzetti** [comment]
I spent good 5 minutes trying to find where to update this template. If we only [listened](https://github.com/gitpod-io/gitpod/pull/7381#issuecomment-1002574247) to @gtsiolis.. 😜

File: .github/workflows/jetbrains-updates-template.yml [hunk 1]
```diff
@@ -59,12 +59,24 @@ jobs:
             This PR updates the ${{ inputs.productName }} IDE image to the latest release version.
 
             ## How to test
-            Go to the preferences and select ${{ inputs.productName }} as desktop IDE. Start a workspace and verify that the new IDE version works.
+            1. Open the preview environment generated for this branch
+            2. Choose ${{ inputs.productName }} as your preferred editor
+            3. Start a workspace using this repository: `https://github.com/gitpod-io/spring-petclinic`
```
Comment (akosyakov) [inline .github/workflows/jetbrains-updates-template.yml:64]: it is only true for IntelliJ, we have automated PRs for all JB supported products

**Comment by akosyakov** [comment]
@andreafalzetti Can we add `team: IDE` label too?

File: .github/workflows/jetbrains-updates-template.yml (hunk 1 above)
Comment (mustard-mh) [reply .github/workflows/jetbrains-updates-template.yml:64]: @andreafalzetti You can make it(repo) a variable and input it from `jetbrains-updates.yml` like others

Comment (akosyakov) [reply .github/workflows/jetbrains-updates-template.yml:64]: Actually it bring a valid point that for each product we should have a repo with different configuration enabled like preconfigured plugins and prebuilds. Maybe for another PR to add such configurations though.

File: .github/workflows/jetbrains-updates-template.yml [hunk 2]
```diff
@@ -59,12 +59,24 @@ jobs:
             This PR updates the ${{ inputs.productName }} IDE image to the latest release version.
 
             ## How to test
-            Go to the preferences and select ${{ inputs.productName }} as desktop IDE. Start a workspace and verify that the new IDE version works.
+            1. Open the preview environment generated for this branch
+            2. Choose ${{ inputs.productName }} as your preferred editor
```
Comment (mustard-mh) [inline .github/workflows/jetbrains-updates-template.yml:63]: It's `stable` one, maybe need to note it too

File: .github/workflows/jetbrains-updates-template.yml [hunk 3]
```diff
@@ -59,12 +59,24 @@ jobs:
...
+
+            - https://www.gitpod.io/docs/troubleshooting#gitpod-logs-in-jetbrains-gateway
+            - https://docs.google.com/document/d/1K9PSB0G6NwX2Ns_SX_HEgMYTKYsgMJMY2wbh0p6t3lQ
 
             ## Release Notes
             ```release-note
             Update ${{ inputs.productName }} IDE image to version ${{ steps.latest-release.outputs.version }}.
             ```
+
+            _This PR was created automatically via [this](https://github.com/gitpod-io/gitpod/blob/main/.github/workflows/jetbrains-updates-template.yml) GitHub Actions workflow_
```
Comment (mustard-mh) [inline .github/workflows/jetbrains-updates-template.yml:79]: It's actually this one JB Releases https://github.com/gitpod-io/gitpod/blob/main/.github/workflows/jetbrains-updates.yml

File: .github/workflows/jetbrains-updates-template.yml [hunk 4]
```diff
@@ -59,12 +59,24 @@ jobs:
...
+            2. Choose ${{ inputs.productName }} as your preferred editor
+            3. Start a workspace using this repository: `https://github.com/gitpod-io/spring-petclinic`
+            4. Verify that the workspace starts successfully
+            5. Verify that the IDE opens successfully
+            6. Verify that the version of the IDE corresponds to the one being updated in this PR
+
+            The following resources should help, in case something goes wrong (e.g. workspaces don't start):
+
+            - https://www.gitpod.io/docs/troubleshooting#gitpod-logs-in-jetbrains-gateway
+            - https://docs.google.com/document/d/1K9PSB0G6NwX2Ns_SX_HEgMYTKYsgMJMY2wbh0p6t3lQ
```
Comment (mustard-mh) [inline .github/workflows/jetbrains-updates-template.yml:72]: Should we make this google doc public?

File: .github/workflows/jetbrains-updates-template.yml (hunk 3 above)
Comment (andreafalzetti) [reply .github/workflows/jetbrains-updates-template.yml:79]: the link goes to the template where text can be updated

Comment (andreafalzetti) [reply .github/workflows/jetbrains-updates-template.yml:79]: I will double-check what to link to

**Comment by andreafalzetti** [comment]
> @andreafalzetti Can we add `team: IDE` label too?

it seems already there https://github.com/gitpod-io/gitpod/pull/9969#event-6597399856 🤔

File: .github/workflows/jetbrains-updates-template.yml (hunk 3 above)
Comment (andreafalzetti) [reply .github/workflows/jetbrains-updates-template.yml]: @mustard-mh I would really like to link to the template, so I've changed the words to be correct. What do you think?

File: .github/workflows/jetbrains-updates-template.yml (hunk 1 above)
Comment (andreafalzetti) [reply .github/workflows/jetbrains-updates-template.yml]: @akosyakov thanks for clarifying, I've updated the wording

File: .github/workflows/jetbrains-updates-template.yml [hunk 5]
```diff
@@ -59,12 +59,24 @@ jobs:
             This PR updates the ${{ inputs.productName }} IDE image to the latest release version.
 
             ## How to test
-            Go to the preferences and select ${{ inputs.productName }} as desktop IDE. Start a workspace and verify that the new IDE version works.
+            1. Open the preview environment generated for this branch
+            2. Choose the stable version of ${{ inputs.productName }} as your preferred editor
+            3. Start a workspace (for IntelliJ use this repository: `https://github.com/gitpod-io/spring-petclinic`)
```
Comment (mustard-mh) [inline .github/workflows/jetbrains-updates-template.yml:64]: something like this `${{ inputs.exampleRepo }}` for now, further see https://github.com/gitpod-io/gitpod/pull/10003#discussion_r872367882

Comment (andreafalzetti) [reply .github/workflows/jetbrains-updates-template.yml:64]: @mustard-mh my understanding is that we don't have example repos for each IDE yet 🤔 Do we?

Comment (mustard-mh) [reply .github/workflows/jetbrains-updates-template.yml:64]: intellij one got some special configurations in `.gitpod.yml` of course😄

File: .github/workflows/jetbrains-updates-template.yml (hunk 3 above)
Comment (mustard-mh) [reply .github/workflows/jetbrains-updates-template.yml]: Up to you 👍

**Review by mustard-mh — APPROVED** [review]
LGTM, feel free to unhold

/hold

File: .github/workflows/jetbrains-updates.yml [hunk 6]
```diff
@@ -22,6 +23,7 @@ jobs:
       productId: goland
       productCode: GO
       productType: release
+      exampleRepo: https://github.com/gitpod-io/gitpod
```
Comment (akosyakov) [inline .github/workflows/jetbrains-updates.yml:26]: https://github.com/gitpod-io/template-golang-cli

File: .github/workflows/jetbrains-updates.yml [hunk 7]
```diff
@@ -31,6 +33,7 @@ jobs:
       productId: pycharm
       productCode: PCP
       productType: release
+      exampleRepo: https://github.com/gitpod-io/gitpod
```
Comment (akosyakov) [inline .github/workflows/jetbrains-updates.yml:36]: https://github.com/gitpod-io/template-python-django

File: .github/workflows/jetbrains-updates.yml [hunk 8]
```diff
@@ -40,5 +43,6 @@ jobs:
       productId: phpstorm
       productCode: PS
       productType: release
+      exampleRepo: https://github.com/gitpod-io/gitpod
```
Comment (akosyakov) [inline .github/workflows/jetbrains-updates.yml:46]: https://github.com/gitpod-io/template-php-laravel-mysql

**Comment by andreafalzetti** [comment]
/unhold

Thanks @mustard-mh @akosyakov 🙏

**Pull request merged**